test-case-reporting/src/setup_db.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require('dotenv').config();
+
+import {Database, dbConnect} from './db';
+
+/** Set up the database table. */
+export async function setupDb(db: Database): Promise<unknown> {
+  return db.schema
+    .createTable('builds', table => {
+      table.increments('id').primary();
+      table.string('commit_hash', 40);
+      table.integer('pr_number');
+      table.dateTime('started_at');
```

Comment:
1. does this have to be a `dateTime` or can this be a [`timestamp`](http://knexjs.org/#Schema-timestamp)? (timestamps are just integers and can be easier to work with for some usecases)
2. does this have to include a timezone? (easier without it)

(asking this about all of the tables)

Reply:
1. It can be a timestamp
2. I don't believe so.
I'll turn them into timestamps & let you know when done.

Reply:
Done.